src/app/fonts/inter.scss

```diff
@@ -0,0 +1,189 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* stylelint-disable max-line-length */
+
+/* Font optimization resources:
+https://web.dev/font-best-practices/
+https://web.dev/css-size-adjust/
+https://barrd.dev/article/create-a-variable-font-subset-for-smaller-file-size/
+https://fonts.googleapis.com/css2?family=Inter:wght@400;600&display=swap
+*/
+
+/* Arial font-size matching for minimal CLS */
+
+@font-face {
+  font-family: Inter-fallback;
```

Comment:
Not relevant to much probably, but I was reading through the Next.js docs the other day and came across this:
https://nextjs.org/docs/pages/building-your-application/optimizing/fonts

```js
import { Inter } from 'next/font/google';
 
// If loading a variable font, you don't need to specify the font weight
const inter = Inter({ subsets: ['latin'] });
```

Not sure if that's worth filing a separate ticket/spike for to investigate in the future if we could potentially do font optimizing.

I also didn't see a Metropolis font on Google Fonts, so wasnt sure how to do potential font optimizing for locally hosted fonts.

Reply:
Yeah, and AFAIK we're already using that - Storybook is useful, but it shouldn't negatively impact what our actual website looks like. So if Storybook has issues loading the fonts, let's see if we can fix that in a way that doesn't require changes to the actual website code - and until we've found such a fix, we'll just have to accept that the fonts don't look quite right there yet.